### DISCUSSION TITLE
MacOS send actual modifier keys to the host.

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#2994] [General]
I use this Mac to connect to my arch box, and the thing is I have i3wm on it will all the custom bindings and on a Mac I have changed my modifier keys because using windows key as a control is just wrong. Is there a way to just send raw keyboard input to the host that I am connecting to. Maybe make an exception to MacOS when I am connected?

**Comment by rustdesk** [comment]
Sadly, no for the time being.

**Reply by ghost** [reply]
Is there a ETA on a fix up because as is it's pretty much non functional as of now?
